sass/layout/_sidebar-left.scss

```diff
@@ -554,33 +639,25 @@
             color: var(--sidebar-text);
         }
 
-        &.draggingOver:hover {
-            //background: radial-gradient(60.83% 3421.87% at 50% 50%, rgba(255, 255, 255, 0.4) 0%, rgba(255, 255, 255, 0) 100%);
-            box-shadow: inset -1px 0 0 var(--sidebar-text-60), inset 0 -1px 0 var(--sidebar-text-60), inset 1px 0 0 var(--sidebar-text-60), inset 0 1px 0 var(--sidebar-text-60);
-            border-radius: 4px;
-        }
-
         i.icon-chevron-down {
             max-width: 16px;
 
             &.hide-arrow {
                 visibility: hidden;
             }
         }
+    }
 
-        .SidebarMenu_menuButton {
-            height: 28px;
-            width: 28px;
-            margin-right: 16px;
-        }
+    .SidebarChannelGroup.menuIsOpen ~ .SidebarChannelGroup .SidebarChannelGroupHeader_groupButton {
```

Comment:
This is another `z-index` issue. Since the menu isn't hoisted outside of the sidebar, the layering of the category header overtop of the menu means that it will always overlap the menu if there is an explicit background.

We need that background to be transparent so that it doesn't overlap the menu, so I've only toggled that while the menu is open.